storage/innobase/xtrabackup/doc/source/howtos/backup_verification.rst

```diff
@@ -24,9 +24,9 @@ After you confirmed that replication has been set up successfully, you can `inst
 
 .. note:: 
 
-  In order for pt-table-checksum to work correctly ``libdbd-mysql-perl`` will need to be installed on *Debian/Ubuntu* systems or ``perl-DBD-MySQL`` on *RHEL/CentOS*. If you installed the *percona-toolkit* package from the Percona repositories package manager should install those libraries automatically.
+  In order for pt-table-checksum to work correctly ``libdbd-mysql-perl`` must be installed on *Debian/Ubuntu* systems or ``perl-DBD-MySQL`` on *RHEL/CentOS*. If you have installed the *percona-toolkit* package from the Percona repositories, those libraries are installed automatically.
```

Comment:
For pt-table-checksum to work correctly, ``libdbd-mysql-perl`` must be installed....

Reply:
Rewritten.